docs/src/conf.py

```diff
@@ -228,7 +228,7 @@ def _dotv(version):
     "matplotlib": ("https://matplotlib.org/stable/", None),
     "numpy": ("https://numpy.org/doc/stable/", None),
     "python": ("https://docs.python.org/3/", None),
-    "scipy": ("https://docs.scipy.org/doc/scipy/", None),
+    "scipy": ("https://docs.scipy.org/doc/", None),
```

Comment:
`scipy` 1.8.0 was released yesterday (sunday 06/02/2022) and there seems to be some issues with their documentation, see https://github.com/scipy/scipy/issues/15540